### PR TITLE
Modified story converter, model, and data

### PR DESCRIPTION
**Pull request by chidiewenike**
## Summary
Modified story_converter.py to train the Rasa Core model with a list of randomly-chosen utterances. Updated the data and uploaded a newly-trained model

## Details
The current story_converter.py creates a list of random utterances to choose from. The update will allow Rasa to randomly choose from this list.

## Testing
Trained a new model on the data generated by story_converter. The model has been included in this PR. The results of the new model can be seen below:

```bash
Your input ->  what can I know about the green house?                                                                                                                                                                            
Originally it was the home of Swiss dairyman, Ambrogio Gianone, who leased it from James Archibald.It is one of the few buildings that survived the 2020 CZU Lightning Fire.
Enjoy your day.
Your input ->  what can I know about the green house?                                                                                                                                                                            
Gianone later bought more land, know as Gianone Hill, where fourth generation children. It is one of the few buildings that survived the 2020 CZU Lightning Fire.
```

### CODE REVIEW

**Review by snekiam — APPROVED** [review]
Looks like there's some merge conflicts on the markdown files. I took a quick look at the diffs and everything seems fine, but they were long and I didn't check them super closely.

**Comment by chidiewenike** [comment]
I think the order of the labels are just different.

**Review by snekiam — APPROVED** [review]

File: rasa-1.x/data/nlu.md [hunk 1]
```diff
@@ -143,6 +288,29 @@
 - Yeah.
 - Okay.
 - I am interested.
+- Yup.
+- Tell me more.
+- Sure.
+- Right on.
```
Comment (Jason-Ku) [inline rasa-1.x/data/nlu.md:294]: How I feel about this PR

**Review by Jason-Ku — APPROVED** [review]

File: rasa-1.x/data/nlu.md (hunk 1 above)
Comment (Jason-Ku) [reply rasa-1.x/data/nlu.md:294]: Oops it was only supposed to get the "Right on." part

**Pull request merged**